.repo/sample-list-generator/README.md

```diff
@@ -0,0 +1,147 @@
+# Sample List Generator
+
+## Overview
+
+It's a script that generates `./extension-samples.json` with the list of all the samples available.
```

Comment:
It would be great to provide some context about why this exists and what it's used for.